src/k8s/pkg/k8sd/features/contour/chart.go

```diff
@@ -33,20 +33,32 @@ var (
 		Namespace:    "projectcontour",
 		ManifestPath: path.Join("charts", "ck-contour-common-1.28.2.tgz"),
 	}
+
+	// registry represents the registry (ghcr.io/canonical) to use for Contour images.
+	registry = "ghcr.io"
```

Comment:
not a huge fan of having this here, then have it be hardcoded in some places and not others. when looking at a `$fooImageRepo` in this package, it is not clear if it should have the `ghcr.io` prefix or not.